packages/playground/src/components/connect_wallet_landing.vue

```diff
@@ -99,6 +97,19 @@ export default {
   setup() {
     const theme = useTheme();
     const baseUrl = import.meta.env.BASE_URL;
+    const network = process.env.NETWORK || (window as any).env.NETWORK;
+
+    const getCapacityHref = () => {
```

Comment:
rename to getCapacityURL(network), your function shouldn't depend on enclosed variables e.g the network const, but make it as a param

Reply:
Better to move into its own file, so it would be easily testable too